modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -38,14 +38,46 @@ $ oc get Network.config.openshift.io cluster -o yaml > cluster-openshift-sdn.yam
 [source,terminal]
 ----
 $ oc patch Network.operator.openshift.io cluster --type='merge' \
-  --patch '{ "spec": { "migration": {"networkType": "OVNKubernetes" } } }'
+  --patch '{ "spec": { "migration": { "networkType": "OVNKubernetes" } } }'
 ----
 +
 [NOTE]
 ====
 This step does not deploy OVN-Kubernetes immediately. Instead, specifying the `migration` field triggers the Machine Config Operator (MCO) to apply new machine configs to all the nodes in the cluster in preparation for the OVN-Kubernetes deployment.
 ====
 
+. Optional: You can enable automatic migration of several OpenShift SDN capabilities to the OVN-Kubernetes equivalents:
```

Comment:
The automatic feature migration is the default setting. Users can disable it if they'd like to.